lochness/mediaflux/__init__.py

```diff
@@ -128,11 +142,11 @@ def sync_module(Lochness: 'lochness.config',
                             subj_dir = subject.protected_folder \
                                 if protect else subject.general_folder
 
-                            # mf_local= pjoin(subj_dir, datatype, dirname(patt), basename(remote))
                             mf_local = str(tree.get(datatype,
                                                 subj_dir,
                                                 processed=processed,
-                                                BIDS=Lochness['BIDS']))
+                                                BIDS=Lochness['BIDS']) / \
+                                                        subpath.parent)
```

Comment:
My commented out line 131 from original Mediaflux design was already doing what you are trying to do now minus the BIDS dichotomy. Here's my [example](https://github.com/AMP-SCZ/lochness/blob/master/docs/source/mediaflux.md#configuration-file) in support of that.
.
Anyways, current line 148 should yield a SyntaxError due to the use of `/`.